### PR TITLE
[Hotfix] UI not handling invalid grafana tokens properly

### PR DESCRIPTION
**Pull request by joeyorlando**
fix issue where frontend would not properly handle scenario where Grafana API token was not yet available/valid

### CODE REVIEW

File: grafana-plugin/src/state/rootBaseStore/index.ts [hunk 1]
```diff
@@ -151,12 +151,12 @@ export class RootBaseStore {
...
     }
 
-    const { allow_signup, is_installed, is_user_anonymous } = pluginConnectionStatus;
+    const { allow_signup, is_installed, is_user_anonymous, token_ok } = pluginConnectionStatus;
     if (is_user_anonymous) {
       return this.setupPluginError(
         '😞 Unfortunately Grafana OnCall is available for authorized users only, please sign in to proceed.'
       );
-    } else if (!is_installed) {
+    } else if (!is_installed || !token_ok) {
```
Comment (iskhakov) [inline grafana-plugin/src/state/rootBaseStore/index.ts:159]: may be add more clear explanation of the error?

**Review by iskhakov — APPROVED** [review]
Approving to hofix the error. Please review that with Michael after he is online

File: Makefile [hunk 2]
```diff
@@ -166,3 +166,6 @@ run-backend-celery:
 
 backend-command:
 	$(call backend_command,$(CMD))
+
+backend-manage-command:
```
Comment (vadimkerr) [inline Makefile:170]: Probably we should remove this from the RP? I propose to only keep stuff that's relevant for the `install` issue

File: grafana-plugin/src/state/rootBaseStore/index.ts (hunk 1 above)
Comment (joeyorlando) [reply grafana-plugin/src/state/rootBaseStore/index.ts:162]: good call. updated the comment for the `setupPlugin` method just a few lines above 👍

**Pull request merged**